package.json

```diff
@@ -243,7 +243,7 @@
     "build-client-if-prod": "node -e \"process.env.CALYPSO_ENV === 'production' && process.exit(1)\" || npm run build-client",
     "build-client-if-desktop": "node -e \"process.env.CALYPSO_ENV === 'desktop' && process.exit(1)\" || npm run build-client",
     "build-packages": "node bin/build-packages.js",
-    "sdk": "node bin/sdk-cli.js",
+    "sdk": "npm run prebuild-css && node bin/sdk-cli.js",
```

Comment:
So this fixes it for `npm run sdk` but it doesn't solve the problem if `node bin/sdk-cli.js` is ran directly (like it is Circle CI)